examples/subquery.cpp

```diff
@@ -1305,6 +1305,22 @@ int main(int, char**) {
                  << endl;
         }
     }
+    {
+        //  SELECT employee_id,first_name,last_name,salary,avg(salary),salary>avg(salary)
+        //  FROM employees;
+        auto rows = storage.select(columns(&Employee::id,
```

Comment:
I bet SQL here is different than
```sql
SELECT employee_id,first_name,last_name,salary,avg(salary),salary>avg(salary)
FROM employees;
```
. I am sure it is
```sql
SELECT employee_id, first_name, last_name, salary, (SELECT avg(salary) FROM employees),salary > (SELECT avg(salary) FROM employees)
FROM employees;
```
Is it right? If it is let's change the comment to make it clear for future readers